src/relax/backend/vm/codegen_vm.cc

```diff
@@ -533,6 +574,10 @@ class CodeGenVM : public ExprFunctor<Instruction::Arg(const Expr&)> {
   const Op& assert_op_ = Op::Get("relax.assert_op");
   const Op& make_closure_op_ = Op::Get("relax.make_closure");
   const Op& invoke_closure_op_ = Op::Get("relax.invoke_closure");
+  const Op& get_captured_cuda_graph_op_ = Op::Get("relax.vm.builtin.get_captured_cuda_graph");
```

Comment:
Can we try to lower directly to `call_builtin` here?

Reply:
see https://github.com/tlc-pack/relax/blob/relax/src/relax/backend/vm/vm_shape_lower.cc#L735